recipes/tensorflow-quantum/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "tensorflow-quantum" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/quantum/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c6a4a72366334673c38cd06329f17889278db4dba8060beb891ef3c489fb72a8
+
+build:
+  number: 0
+  skip: true   # [ py<36 or win]
+  script:
+    - export TF_NEED_CUDA=0
+    - export TF_CUDA_VERSION=10.1
+    - ./configure.sh
+    - bazel build -c opt --cxxopt="-O3" --cxxopt="-march=native" --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" release:build_pip_package
```

Comment:
The options here will lead to packages that won't run except on the CI machines themselves. Can you use the bazel-toolchain package to generate matching flags?

Reply:
Thai is from upstream. Will try to use the toolchain to generate these.

Reply:
`bazel-toolchain` imposes `bazel >=4` and currently the OSX build [fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=432757&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=2845). According to https://github.com/bazelbuild/bazel/issues/12887 this might be connected so I'll have to investigate 1) if this is actually the cause and 2) if there is a workaround.

Reply:
Hitting the same error on Linux now. I think it's a combination of different versions that's not supported.